data/channels.csv

```diff
@@ -21986,6 +22000,7 @@ StarMoviesIndia.in,Star Movies India,,,,IN,,,c/IN,hin,movies,FALSE,2011-04-01,,,
 StarMoviesMiddleEast.ae,Star Movies Middle East,,,,AE,,,r/MIDEAST,ara,movies,FALSE,,,,http://foxarabia.tv/,https://i.imgur.com/JhT3k6R.png
 StarMoviesPlus.tz,Star Movies +,,,,TZ,,,c/TZ,eng,movies,FALSE,,,,,https://i.imgur.com/ljxL3O6.png
 StarMoviesSelect.in,Star Movies Select,,,,IN,,,c/IN,hin,movies,FALSE,2015-07-09,,,http://www.starmovies.in/,https://i.imgur.com/UomTc3O.png
+Starnet.in,Starnet,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/bYqsCr1.png

```

Comment:
```suggestion
Starnet.in,Starnet,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/J3jB7Kr.png
```
Removed the transparent background around the logo.

Reply:
The original image already has a transparent background!